dss/operations/sync.py

```diff
@@ -26,6 +29,42 @@ def _log_warning(**kwargs):
     logger.warning(json.dumps(kwargs))
 
 
+@sync.action('verify-sync-all',
+             arguments={"--source-replica": {'choices': [r.name for r in Replica], 'required': True},
+                        "--destination-replica": {'choices': [r.name for r in Replica], 'required': True},
+                        "--since": {'required': False, 'help': "Only check objects newer than this (DSS_VERSION)"}})
+def verify_sync_all(argv: typing.List[str], args: argparse.Namespace):
+    """
+    Like verify-sync, but provides keys from the designated source replica
+    (so that keys don't need to be supplied individually).
+    """
+    def list_objects_since(replica, since: datetime.datetime):
+        replica = Replica[replica]
+        handle = Config.get_blobstore_handle(replica)
+        for prefix in (BUNDLE_PREFIX, FILE_PREFIX, BLOB_PREFIX):
+            for name, metadata in handle.list_v2(replica.bucket, prefix=prefix):
+                if metadata[BlobMetadataField.LAST_MODIFIED] > since:
```

Comment:
might want to use .get(), can LM values be unset?

Reply:
`Last-Modified` is provided by S3 for every object. I think it makes sense for this code to fail if that data is missing - without it, we can't determine if it should be checked, and I think that is a problem worth breaking for (as opposed to always skipping or always verifying).